components/Button/NavButton.tsx

```diff
@@ -0,0 +1,26 @@
+"use client";
+import Button from "@/components/Button/Button";
+import { ButtonType } from "@/components/Button/utils";
+import { useRouter } from "next/navigation";
+
+const NavButton = ({
+  className,
+  buttonType,
+  link,
+}: {
+  className?: string;
+  buttonType?: ButtonType;
+  link?: string;
+}) => {
+  const router = useRouter();
+  return (
+    <Button
+      label="Get started"
+      buttonType={buttonType || ButtonType.NAVIGATION}
```

Comment:
use `??`instead of  `||` for default values it is correct semantically and decrease potential issues with empty strings, 0 etc..